packages/oui-navbar/src/keyboard-keys.constant.js

```diff
@@ -1,5 +1,5 @@
 export default {
-    ALT: 9,
-    TAB: 16,
+    TAB: 9,
```

Comment:
Could be used elsewhere, don't we have a package for this kind of universal code ?